util/test/testUtil.h

```diff
@@ -44,7 +45,7 @@ void valuecheck(const Eigen::DenseBase<DerivedA>& a, const Eigen::DenseBase<Deri
 
 void valuecheck(double a, double b, double tolerance = 1e-8)
 {
-  if (abs(a - b) > tolerance) {
+  if (std::abs(a - b) > tolerance) {
```

Comment:
oops. it's still waiting in my other pull request (which I hope to get to today)